LICENSE

```diff
@@ -0,0 +1,277 @@
+Eclipse Public License - v 2.0
```

Comment:
Hi @lauren-li, I am not sure why this file added because I didn't add it. 🤔

Reply:
@tiantn No worries, you can try the solution below to remove it: https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request#answer-44355701

Reply:
Thank you @lauren-li . I followed the instrctions to remove it. Thank you!

Reply:
Hi @tiantn, thank you for your efforts on this! However, I am still seeing the LICENSE file in the PR. Could you try again, please?

Reply:
Hi @lauren-li , I still can't remove it. From the following graph, the file is not added by me. Is this a reason for removing failed? 
![图片](https://user-images.githubusercontent.com/23715327/150738568-6d956e96-2fa9-4ea0-87b7-9d075f22ce29.png)

Reply:
Hi @tiantn, I also gave this a try, and I also failed to remove it. 😞 However, when I checked out the `master` branch's LICENSE file into your branch, Git showed no changes, so I guess it will have no effect and we can leave this as-is. Thank you for trying to remove it from the PR, though!